src/utils/rest-api.ts

```diff
@@ -527,6 +527,30 @@ export function getContingencyList(type: string, id: string) {
     });
 }
 
+/**
+ * Get the basic data of the network modifications contained in a composite modification
+ */
+export function getCompositeModificationContent(id: string) {
```

Comment:
fetchNetworkCompositeModification

Reply:
OK for fetch but this is the content of the composite modification that we are fetching, not the composite modification itself, which we already have.